apps/mannschaft/src/mannschaft-content-box.tsx

```diff
@@ -0,0 +1,45 @@
+import {styled} from '@mui/material'
+import {
+  BuilderEmbedBlockProps,
+  BuilderImageBlockProps,
+  BuilderRichTextBlockProps
+} from '@wepublish/website'
+import {useWebsiteBuilder} from '@wepublish/website'
+import {ApiV1} from '@wepublish/website'
+import {allPass, anyPass} from 'ramda'
+import {hasBlockStyle} from '@wepublish/website'
+import {isImageBlock} from '@wepublish/website'
+import {isRichTextBlock} from '@wepublish/website'
+import {isEmbedBlock} from '@wepublish/website'
+
+export const MannschaftContentBox = (
+  props: BuilderImageBlockProps | BuilderRichTextBlockProps | BuilderEmbedBlockProps
+) => {
+  const {
+    blocks: {Image, RichText, Embed}
+  } = useWebsiteBuilder()
+
+  return (
+    <MannschaftContentBoxWrapper>
+      {isImageBlock(props) && <Image {...props} />}
+      {isRichTextBlock(props) && <RichText {...props} />}
+      {isEmbedBlock(props) && <Embed {...props} />}
+    </MannschaftContentBoxWrapper>
+  )
+}
+
+const MannschaftContentBoxWrapper = styled('div')`
+  padding: ${({theme}) => theme.spacing(3)};
+  background-color: lightyellow;
```

Comment:
I think the color should come from the theme, we already have a yellow there (accent light)